docs/upgrade.html

```diff
@@ -39,7 +39,7 @@ <h5><a id="upgrade_280_notable" href="#upgrade_280_notable">Notable changes in 2
     </li>
 </ul>
 
-<h4><a id="upgrade_2_7_0" href="#upgrade_2_7_0">Upgrading to 2.7.0 from any version 0.8.x through 2.6.x</a></h4>
+<h4><a id="upgrade_2_8_1" href="#upgrade_2_8_1">Upgrading to 2.8.1 from any version 0.8.x through 2.7.x</a></h4>
```

Comment:
looks like we missed adding `Upgrading to 2.8.0` section earlier. I think, we should add new section from "2.8.0"  release and retain existing "Upgrading to 2.7.0" section.